### PR TITLE
fixes troubleshooting-kubeadm references to taint node-role.kubernetes.io/master

### PR DESCRIPTION
**Pull request by marshallford**
The `role.kubernetes.io/master` taint referenced on the troubleshooting-kubeadm page should be `node-role.kubernetes.io/master`.

Please let me know if I should remove the other lang changes. I'm not sure what the procedure is for code snippets vs actual translations. Thanks.

### CODE REVIEW

File: content/en/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md [hunk 1]
```diff
@@ -307,17 +307,17 @@ The tracking issue for this problem is [here](https://github.com/kubernetes/kube
...
 
 There are at least two workarounds:
 
-1. Use the `role.kubernetes.io/master:PreferNoSchedule` taint instead of an empty slice. [Pods will get scheduled on masters](/docs/concepts/configuration/taint-and-toleration/), unless other nodes have capacity.
+1. Use the `node-role.kubernetes.io/master:PreferNoSchedule` taint instead of an empty slice. [Pods will get scheduled on masters](/docs/concepts/configuration/taint-and-toleration/), unless other nodes have capacity.
 
 2. Remove the taint after kubeadm init exits:
 ```bash
-kubectl taint nodes NODE_NAME role.kubernetes.io/master:NoSchedule-
+kubectl taint nodes NODE_NAME node-role.kubernetes.io/master:NoSchedule-
```
Comment (neolit123) [inline content/en/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md]: the fix is valid, thanks.
but AFAIK a PR should only touch a single language - e.g. `en`.

Comment (marshallford) [reply content/en/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md]: Fixed.

**Comment by marshallford** [comment]
Do I need all three reviewers because I touched the other langs and then removed the changes?

**Comment by neolit123** [comment]
no, i don't think so, even if the labels remained.
/assign @gochist

try asking on #sig-docs on k8s slack if no approval is reached until e.g. mid next week.

**Comment by tanjunchen** [comment]
/cc 
i will sycn th zh once this merged

**Comment by tanjunchen** [comment]
/lgtm
kindly /cc @sftim

**Pull request merged**